test/functions.jl

```diff
@@ -163,7 +163,13 @@
             @test MOIU.operate(+, Float64, MOI.SingleVariable(x),
                                MOI.SingleVariable(z)) + 1.0 ≈
                   MOI.ScalarAffineFunction(MOI.ScalarAffineTerm.([1, 1e-7, 1], [x, y, z]), 1.0) atol=1e-6
-            @test MOI.ScalarAffineFunction([MOI.ScalarAffineTerm(1.0, x), MOI.ScalarAffineTerm(1e-7, y)], 1.0) ≈ MOI.ScalarAffineFunction([MOI.ScalarAffineTerm(1, x)], 1) atol=1e-6
+            f1 = MOI.ScalarAffineFunction([MOI.ScalarAffineTerm(1.0, x), MOI.ScalarAffineTerm(1e-7, y)], 1.0)
```

Comment:
It's hard to see what this `testset` is testing. It would be better to have smaller testsets that test individual operators.